test/unit/component.js

```diff
@@ -590,3 +590,27 @@ test('should provide interval methods that automatically get cleared on componen
 
   ok(intervalsFired === 5, 'Interval was cleared when component was disposed');
 });
+
+test('should ignore case in language codes and try primary code', function() {
+  expect(3);
+
+  var player = PlayerTest.makePlayer({
+    'languages': {
+      'en-gb': {
+        'Good': 'Brilliant'
+      },
+      'EN': {
+        'Good': 'Awesome',
+        'Error': 'Problem'
+      }
+    }
+  });
+
+  var comp = new vjs.Component(player);
+
+  player.language('en-gb');
+  ok(comp.localize('Good') === 'Brilliant', 'Used subcode specific localization');
```

Comment:
You can use `strictEqual()` (for `===` equality) or `equal()` (for `==`) instead of `ok()` to get better error messages when these assertions fail.